backend/src/services/user-details/user-details.service.ts

```diff
@@ -34,9 +34,19 @@ class UserDetails {
     userId: number,
     userDetailsUpdateInfoRequestDto: UserDetailsUpdateInfoRequestDto,
   ): Promise<UserDetailsResponseDto | null> {
+    const hasTelegram = Boolean(
+      userDetailsUpdateInfoRequestDto.telegramUsername,
+    );
+    const { telegramUsername } = userDetailsUpdateInfoRequestDto;
```

Comment:
```suggestion
    const { telegramUsername } = userDetailsUpdateInfoRequestDto;
    const hasTelegram = Boolean(telegramUsername);
```
why not just use extracted value for boolean check